tpot/base.py

```diff
@@ -85,6 +85,7 @@ class TPOTBase(BaseEstimator):
 
     # dont save periodic pipelines more often than this
     OUTPUT_BEST_PIPELINE_PERIOD_SECONDS = 30
+    MAX_MUT_LOOPS = 50
```

Comment:
Please refactor both of these constants to internal variables (with a `_` in front).

I know your PR doesn't add `OUTPUT_BEST_PIPELINE_PERIOD_SECONDS`, but might as well fix both here.